### PR TITLE
Example

### PR DESCRIPTION
**Pull request by EllairaT**

### CODE REVIEW

File: components/Button.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function Button(props) {
+    console.log("hello")
```
Comment (fengv1976) [inline components/Button.js:2]: test

**Review by fengv1976 — CHANGES_REQUESTED** [review]

File: components/Button.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 function Button(props) {
-    console.log("hello")
+    console.log('hello')
```
Comment (EllairaT) [inline components/Button.js:2]: changed it according to victor's feedback

Comment (fengv1976) [reply components/Button.js:2]: good

**Review by fengv1976 — APPROVED** [review]
good

**Pull request closed without merging**